app/src/components/common/TmSessionExtension.vue

```diff
@@ -58,7 +58,13 @@ export default {
     ...mapState([`extension`]),
     ...mapGetters([`networks`]),
     accounts() {
-      return this.extension.accounts
+      // filter forgotten accounts
+      const forgottenAccountsList = localStorage.getItem(
```

Comment:
please remove actually

Reply:
there is already a 'DELETE_WALLET' handler in the messageHandlers.js

Reply:
oh, is there? Wow, then this was only missing the visual part. The logic was already there